--- pycdio-0.20/swig/device_const.swg.orig	2013-02-20 12:32:41.000000000 +0100
+++ pycdio-0.20/swig/device_const.swg	2017-12-10 20:53:27.839959959 +0100
@@ -83,7 +83,6 @@
 /* driver_id_t enums. */
 %constant long int DRIVER_UNKNOWN = DRIVER_UNKNOWN;
 %constant long int DRIVER_AIX     = DRIVER_AIX;
-%constant long int DRIVER_BSDI    = DRIVER_BSDI;
 %constant long int DRIVER_FREEBSD = DRIVER_FREEBSD;
 %constant long int DRIVER_LINUX   = DRIVER_LINUX;
 %constant long int DRIVER_SOLARIS = DRIVER_SOLARIS;
diff -ur pycdio-0.20.orig/cdio.py pycdio-0.20/cdio.py
--- pycdio-0.20.orig/cdio.py	2018-02-15 18:19:00.344688689 +0100
+++ pycdio-0.20/cdio.py	2018-02-15 18:19:09.607963138 +0100
@@ -47,7 +47,6 @@
 drivers = {
     'Unknown'  : pycdio.DRIVER_UNKNOWN,
     'AIX'      : pycdio.DRIVER_AIX,
-    'BSDI'     : pycdio.DRIVER_BSDI,
     'FreeBSD'  : pycdio.DRIVER_FREEBSD,
     'GNU/Linux': pycdio.DRIVER_LINUX,
     'linux'    : pycdio.DRIVER_LINUX,
diff -ur pycdio-0.20.orig/pycdio.py pycdio-0.20/pycdio.py
--- pycdio-0.20.orig/pycdio.py	2018-02-15 18:19:00.344688689 +0100
+++ pycdio-0.20/pycdio.py	2018-02-15 18:19:16.171254642 +0100
@@ -653,7 +653,6 @@
 SRC_IS_NATIVE_MASK = _pycdio.SRC_IS_NATIVE_MASK
 DRIVER_UNKNOWN = _pycdio.DRIVER_UNKNOWN
 DRIVER_AIX = _pycdio.DRIVER_AIX
-DRIVER_BSDI = _pycdio.DRIVER_BSDI
 DRIVER_FREEBSD = _pycdio.DRIVER_FREEBSD
 DRIVER_LINUX = _pycdio.DRIVER_LINUX
 DRIVER_SOLARIS = _pycdio.DRIVER_SOLARIS
