--- pycdio-0.20/swig/device_const.swg.orig	2013-02-20 12:32:41.000000000 +0100
+++ pycdio-0.20/swig/device_const.swg	2017-12-10 20:53:27.839959959 +0100
@@ -83,7 +83,6 @@
 /* driver_id_t enums. */
 %constant long int DRIVER_UNKNOWN = DRIVER_UNKNOWN;
 %constant long int DRIVER_AIX     = DRIVER_AIX;
-%constant long int DRIVER_BSDI    = DRIVER_BSDI;
 %constant long int DRIVER_FREEBSD = DRIVER_FREEBSD;
 %constant long int DRIVER_LINUX   = DRIVER_LINUX;
 %constant long int DRIVER_SOLARIS = DRIVER_SOLARIS;
